src/validation.cairo

```diff
@@ -350,4 +439,39 @@ mod tests {
         let last_reward = compute_block_reward(max_halvings * block_height);
         assert_eq!(last_reward, 0);
     }
+    // #[test]
```

Comment:
Why is it commented out?

Reply:
not working for now with correct data. Still working on it, trying to find the right way to have the correct input in bytes representation